Mh/rc to play pt2

bleehu opened the pull request:
Round 2. Shaved it down to 230 commits.

bleehu committed:
added armor peircing levels to medic attacks

bleehu committed:
Balanced Armor piercing levels for medic proceedures. Also spell checked.

bleehu committed:
Changed to-miss percentages to conform with Combat 2.0 standards

bleehu committed:
started work removing nanites from engineering

bleehu committed:
buffed last stand

bleehu committed:
added constituent elements of engineer processes

bleehu committed:
added engineer's consumable items and put limits on engineer's elements.

bleehu committed:
added level progression limits and removed nanite gen/cost

bleehu committed:
transitioned several engineer processes to use energy and material instead of nanites

bleehu committed:
removed Nanite cost from Last stand

bleehu committed:
Merge branch 'dev' into MH-Last-Stand-Feat

bleehu committed:
added the -50 hp rule suggested by 1sourcecontrol

bleehu committed:
clarified ambiguity in class feat acquisition

bleehu committed:
Reverted to new Gunkata

bleehu committed:
Rounded half accuracy modifiers

bleehu committed:
added explicit action limit and added when the feat is activated.

bleehu committed:
Merge branch 'dev' into MH-Issue-174-Class-Feat-Ambiguity
Removed Class feat rank up language and prioritized Skills link over rule clarification

 Conflicts:
	CharacterCreation/02_Classes.txt

 Changes to be committed:
	modified:   02_Classes.txt
	modified:   03_Feats.txt
	modified:   05_ArmourList.txt
	modified:   Class-Specific Documentation/Medic Proceedures.txt
	modified:   Class-Specific Documentation/Medic Skill Tree - Malpractice.png

bleehu committed:
added Proficency rules and glossary entries

bleehu committed:
spell checked Races

bleehu committed:
clean up partial work for separate commit

bleehu committed:
removed old Quality RP stat

bleehu committed:
some more fixes

bleehu committed:
Merge branch 'dev' into MH-update-Engineer

 Conflicts:
	CharacterCreation/02_Classes.txt

 Changes to be committed:
	modified:   ../Basic Rules.txt
	modified:   02_Classes.txt
	modified:   03_Feats.txt
	renamed:    04_02_WeaponAttachments.txt -> 04_00_WeaponAttachments.txt
	new file:   04_01_Melee_&_Thrown_Weapons.txt
	deleted:    04_01_Weapons.txt
	new file:   04_02_Pistols.txt
	deleted:    04_03_Advanced Weapons.txt
	new file:   04_03_SMGs_&_Shotguns.txt
	new file:   04_04_Carbines_AssaultRifles_&_BattleRifles.txt
	new file:   04_05_Long_Rifles_&_DMRs.txt
	new file:   04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt
	modified:   05_ArmourList.txt
	modified:   06_Skills.txt
	deleted:    Character Sheets/Daeron_CharSheet.txt
	deleted:    Character Sheets/Monk_Hirito_12.txt
	deleted:    Character Sheets/Paladin_Tibor_6.txt
	deleted:    Character Sheets/[TEMPLATE](CHARNAME)_CharSheet.txt
	modified:   Class-Specific Documentation/Medic Proceedures.txt
	modified:   Class-Specific Documentation/Medic Skill Tree - Malpractice.png
	modified:   Class-Specific Documentation/Tome of Spells.txt
	new file:   ../Glossary_of_Terms.txt

bleehu committed:
Merge pull request #199 from bleehu/MH-Issue-174-Class-Feat-Ambiguity

Fix Issue #174.

bleehu committed:
Merge pull request #211 from bleehu/MH-Issue-206-RP-Races

removed old Quality RP stat

bleehu committed:
Merge pull request #179 from bleehu/MH-Last-Stand-Feat

MH Last Stand

1sourcecontrol committed:
Update 04_00_WeaponAttachments.txt

1sourcecontrol committed:
Update 04_00_WeaponAttachments.txt

1sourcecontrol committed:
Update Basic Rules.txt

1sourcecontrol committed:
held actions and reactions in the glossary

1sourcecontrol committed:
turtle catch

1sourcecontrol committed:
spellcheck & modified melee rules & added definition of Opportunity Attacks

1sourcecontrol committed:
added an example, per Michael's comment

bleehu committed:
Fort changes

1sourcecontrol committed:
added opportunity attack to the glossary

bleehu committed:
Merge branch 'dev' into MH-Mr-Handy

bleehu committed:
Merge pull request #208 from bleehu/MH-Mr-Handy

Races document spell check

1sourcecontrol committed:
Merge pull request #215 from bleehu/pistol-quickdraw-interaction-fix_-alexL

Update pistol lanyard with pistol quickdraw change

1sourcecontrol committed:
Merge pull request #216 from bleehu/add-reactions_held_reactions_AlexL

reaction_held reactions addition

Turtlelord26 committed:
Favored Class Text Deleted

Also aligned Opaleite text to match the rest of the file.

Turtlelord26 committed:
Add Pointer to New Player Walkthrough

Turtlelord26 committed:
Create 00_New_Player_Walkthrough.txt

Still WIP, waiting on scheduled fundamental game updates.

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

Markdown is a thing that requires specific syntax.

1sourcecontrol committed:
Basic Rules conformity to the style guide

1sourcecontrol committed:
Update Basic Rules.txt

1sourcecontrol committed:
kill last stand duplicate

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

Addressing bleehu review.

1sourcecontrol committed:
Update 04_02_Pistols.txt

This pistol is (aesthetically) far off from the other weapons, adjusting to make more realistic/immersive sense.

1sourcecontrol committed:
Merge pull request #220 from bleehu/kls-duplicate-AlexL

kill last stand duplicate

1sourcecontrol committed:
michael nanites comment

1sourcecontrol committed:
Merge pull request #219 from bleehu/basicrules_style_edit_AlexL

Basic Rules conformity to the style guide and other

Turtlelord26 committed:
Space Created

I can hardly complain when you nitpick me given how much I nitpick everyone else.

1sourcecontrol committed:
Merge pull request #221 from bleehu/pistol_minorpatch_AlexL

Update 04_02_Pistols.txt, minor patch

1sourcecontrol committed:
Update Basic Rules.txt

1sourcecontrol committed:
changes suggested by turtle

and a little more

1sourcecontrol committed:
Merge pull request #225 from bleehu/modify-prone-rules_AlexL

Modify prone rules

bleehu committed:
change races to have the agreed upon style of headers

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

1sourcecontrol committed:
Update Glossary_of_Terms.txt

added a file "chapter" heading

1sourcecontrol committed:
styleguide

1sourcecontrol committed:
Update Engineer Processes.txt

styleguide update and spelling fixes

bleehu committed:
Stylized Races and added some headings

1sourcecontrol committed:
Update Hacking.txt

styleguide edits

bleehu committed:
Modified Classes to conform to plaintext style guides

1sourcecontrol committed:
Update Medic Proceedures.txt

styleguide edits

1sourcecontrol committed:
Update 03_Feats.txt

styleguide initial pass

1sourcecontrol committed:
Update 04_00_WeaponAttachments.txt

styleguide inital pass

1sourcecontrol committed:
Update 04_01_Melee_&_Thrown_Weapons.txt

styleguide initial edits

1sourcecontrol committed:
Update 04_02_Pistols.txt

styleguide initial edits

1sourcecontrol committed:
Update 04_03_SMGs_&_Shotguns.txt

styleguide initial

1sourcecontrol committed:
Update 04_02_Pistols.txt

1sourcecontrol committed:
Update 04_04_Carbines_AssaultRifles_&_BattleRifles.txt

styleguide inital edits

1sourcecontrol committed:
Update 04_05_Long_Rifles_&_DMRs.txt

styleguide initial edit

1sourcecontrol committed:
Update 04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

styleguide initial edit

1sourcecontrol committed:
Update 05_ArmourList.txt

styleguide inital edits

1sourcecontrol committed:
Update 07_Items.txt

styleguide initial edit

1sourcecontrol committed:
Update 06_Skills.txt

styleguide initial edit

1sourcecontrol committed:
Update 09_Tactical Movement.txt

Styleguide Initial edits

Turtlelord26 committed:
Update 00_New_Player_Walkthrough.txt

credits > money

Turtlelord26 committed:
Merge pull request #218 from bleehu/Turtlelord26-Issue207-Remove_Favored_Classes

New Player Guide

1sourcecontrol committed:
Merge pull request #229 from bleehu/soul_sucking_edits_AlexL

Initial styleguide pass through the used files

bleehu committed:
some small cleanups. Some lists I missed.

bleehu committed:
Changed backtick character to ascii-compliant apostrophe

bleehu committed:
one more backtick

bleehu committed:
Merge branch 'dev' into MH-Issue-201-Profficency-Glossary

 Conflicts:
	Basic Rules.txt
	Glossary_of_Terms.txt

bleehu committed:
grammar cleanup of Proficency language

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

 Conflicts:
	CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

bleehu committed:
added section heading quotes

bleehu committed:
added flavor note to free running.

1sourcecontrol committed:
Update Skill DCs.txt

copyright text removal

bleehu committed:
Removed unused Nanite Push Proceedure

1sourcecontrol committed:
Update Basic Rules.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update Engineer Processes.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update Medic Proceedures.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 01_Races.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 02_Classes.txt

1sourcecontrol committed:
Update 03_Feats.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 04_00_WeaponAttachments.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 05_ArmourList.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 06_Skills.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 07_Items.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update README.txt

Remove copyright text at the eof

1sourcecontrol committed:
Update 07_Items_L10.txt

Remove copyright text at the eof

bleehu committed:
grammar check, scaled Harmacist spike and fixed shield penetration language

bleehu committed:
stylized Medic Proceedures, added payload clarification and fixed some language on defeating armor

bleehu committed:
Merge pull request #212 from bleehu/Mike-Update-Basic-Rules

modify push/shove

bleehu committed:
Merge branch 'dev' into MH-Issue-214-Quotes-Glossary

bleehu committed:
some styling and added quotes to section reference in basic rules

bleehu committed:
Merge pull request #235 from bleehu/MH-Issue-214-Quotes-Glossary

Mh issue 214 quotes glossary

bleehu committed:
removed sutter

bleehu committed:
clarified Berserker unarmed expert

bleehu committed:
Merge pull request #227 from bleehu/MH-Stylize-Classes

Modified Classes to conform to plaintext style guides

1sourcecontrol committed:
Merge pull request #240 from bleehu/remove_extran_copyrighttxt_1source

merge to dev - removed copyright notice

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

bleehu committed:
spell procedures like the dictionary does

bleehu committed:
updated intro paragraph

bleehu committed:
Medic isn't a class. Renamed Document

bleehu committed:
It's not called Medic, it's called doctor. Also Changed "Quarentine" to "Contaminate Area".
Added some style changes.
 Changes to be committed:
	renamed:    Medic Skill Tree - Hippocratic.png -> Doctor Procedure Tree - Hippocratic.png
	new file:   Doctor Procedure Tree - Malpractice.png
	modified:   Doctor Procedures.txt
	deleted:    Medic Skill Tree - Malpractice.png

bleehu committed:
finished stylizing the Proceedures Doc

bleehu committed:
Merge pull request #241 from bleehu/MH-Issue-201-Profficency-Glossary

removed sutter

bleehu committed:
added whateverectomy

1sourcecontrol committed:
styleguide conformity

1sourcecontrol committed:
styleguide_2nd_passthrough

1sourcecontrol committed:
modify punch drunk

Change to C2.0 miss chance rounding.

1sourcecontrol committed:
addec colons, spellchecked

1sourcecontrol committed:
Merge pull request #246 from bleehu/punchdrunk_c2.0_AL

merge to dev; modify punch drunk

1sourcecontrol committed:
adding 'Prerequisites: None'

1sourcecontrol committed:
Bleehu tweeks

1sourcecontrol committed:
fix feat name issues

1sourcecontrol committed:
armor names

A little scope creep for this PR, the armor names weren't updated to C2.0 standards

1sourcecontrol committed:
Merge branch 'dev' into styleguide_2_AL

1sourcecontrol committed:
Merge pull request #245 from bleehu/styleguide_2_AL

styleguide_for_feats

1sourcecontrol committed:
Merge pull request #244 from bleehu/styleguid_AL

styleguide conformity; armorlist

1sourcecontrol committed:
Update 05_ArmourList.txt

1sourcecontrol committed:
Update 05_ArmourList.txt

missed a spot

1sourcecontrol committed:
update APs to AS and grammer/styleguide

bleehu committed:
Spell check and copywrite removal.

bleehu committed:
added space for consistency

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

also cleaned up racial traits

 Conflicts:
	CharacterCreation/01_Races.txt

bleehu committed:
Stylized Basic Rules

bleehu committed:
Added Free Action to Glossary

bleehu committed:
some small typos

bleehu committed:
updated buzzed to conform to Combat 2.0

bleehu committed:
Merge pull request #252 from bleehu/MH-Issue-251-Buzzed2.0

updated buzzed to conform to Combat 2.0

1sourcecontrol committed:
Merge pull request #248 from bleehu/address#198_nStyleGuide_AL

merge to dev,  for issue #198. Styleguide

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

bleehu committed:
bumped version number

bleehu committed:
removed names

bleehu committed:
Some simple cleanup

bleehu committed:
Stylized Items document.

bleehu committed:
Merge pull request #255 from bleehu/MH-version-beta-0.0.2-bump

bumped version number

bleehu committed:
Some grammar and capitolization details cleaned up

bleehu committed:
Lowered prices on Anfo and breaching charges

bleehu committed:
removed unnecessary section header from engineering processes

bleehu committed:
Merge pull request #257 from bleehu/MH-Stylize-Engineer-Processes

Stylize Engineer Process Docs

bleehu committed:
removed old pre-beta sheet

bleehu committed:
Stylized Basic Rules to conform with proposed table styles

bleehu committed:
Merge pull request #260 from bleehu/MH-remove-old-sheet

removed old pre-beta sheet

bleehu committed:
added glow sticks to items

Turtlelord26 committed:
Language Deletion Pending Update

re #259

bleehu committed:
Merge pull request #202 from bleehu/MH-medic-2.0

Mh medic 2.0

Turtlelord26 committed:
Merge pull request #266 from bleehu/Turtlelord26-Temp_Languages_Removal

Language Deletion Pending Update

bleehu committed:
added glowsticks but for real

bleehu committed:
Merge pull request #261 from bleehu/MH-Basic-Table-Styles

Stylized Basic Rules to conform with proposed table styles

bleehu committed:
removed copywrite compound from glowsticks

just to be safe.

bleehu committed:
Update repair kit to match med kit

why use two paragraphs when one will do?

bleehu committed:
Merge pull request #263 from bleehu/MH-add-Glowsticks

Added glow sticks to items

Turtlelord26 committed:
Stylize New Player Walkthrough

re Issue #236

Turtlelord26 committed:
Merge pull request #267 from bleehu/Turtlelord26-Stylize_Walkthrough

Stylize New Player Walkthrough

Turtlelord26 committed:
Skills doc draft from hulk

Turtlelord26 committed:
Add files via upload

Turtlelord26 committed:
Readability enabled

Turtlelord26 committed:
Rename Skills Draft.txt to 06_01_Systemized_Skills.txt

Turtlelord26 committed:
Merge pull request #268 from bleehu/Turtlelord26-skillsdraft

Skills doc draft from hulk

bleehu committed:
Merge branch 'dev' into MH-Stylize-Races

Turtlelord26 committed:
Update 07_Items.txt

Turtlelord26 committed:
Addressing Racial Stylization Review

Handled all blocking and nonblocking issues from my earlier review, except for the inconsistent handling of racial trait style. I didn't want to unilaterally pick a way to do that, as I'm not on the team responsible for generating the traits.

bleehu committed:
took out the trash in races doc

Turtlelord26 committed:
Merge pull request #282 from bleehu/Turtlelord26-Stylize-Races_Review

Addressing Racial Stylization Review

bleehu committed:
Merge pull request #226 from bleehu/MH-Stylize-Races

Stylized Races document

Turtlelord26 committed:
Addressing Basic Rules Stylization Review

Fixed all blockers and nonblockers from my recent review, except tables.

Turtlelord26 committed:
Merge pull request #281 from bleehu/Turtlelord26-Stylize-Items_Review

Addressing Items Stylization Review

bleehu committed:
ordered top directory and added conditions file

bleehu committed:
added and styled conditions

bleehu committed:
Merge branch 'dev' into MH-Stylize-Items

bleehu committed:
Merge pull request #258 from bleehu/MH-Stylize-Items

Stylized Items document

bleehu committed:
added hazmat suits

Turtlelord26 committed:
Partial Reversion

bleehu committed:
Merge pull request #285 from bleehu/MH-Issue-277-Hazmats

added hazmat suits

Turtlelord26 committed:
Merge pull request #284 from bleehu/Turtlelord26-Glossary-Style-Fixes_Review

Addressing Basic Rules Stylization Review

bleehu committed:
Merge branch 'dev' into MH-Glossary-Style-Fixes

branch caught it comittee while other work is being done. Need to not squash that work.

bleehu committed:
Merge pull request #250 from bleehu/MH-Glossary-Style-Fixes

Glossary  and Basic Rules style fixes

bleehu committed:
terminated table with a newline

bleehu committed:
Merge branch 'dev' into MH-conditions-section

Reveraine committed:
Added Roll Definitions to Glossary

Average rolling was something I thought of while reworking races. I don't think it's too complex, but it will need some testing, I think. I'll put more thoughts for it in basic rules.

Reveraine committed:
Rolling the Average to Base Rules

bleehu committed:
removed redundant conditions section

Reveraine committed:
Added reworked Tundari

There are a few things missing. What a weakness to poison looks like and STAT bonuses are not in yet.

Reveraine committed:
Update Glossary_of_Terms.txt

Reveraine committed:
Merge pull request #287 from bleehu/Reveraine-patch-1

Added Roll Definitions to Glossary

Reveraine committed:
Merge pull request #288 from bleehu/Reveraine-patch-2

Rolling the Average to Base Rules

bleehu committed:
Merge branch 'dev' into MH-conditions-section

bleehu committed:
Merge pull request #286 from bleehu/MH-conditions-section

Mh conditions section

Reveraine committed:
Update 01_03_Races_Tundari

Reveraine committed:
Merge pull request #289 from bleehu/Reveraine-Race-Updates

Added reworked Tundari

Turtlelord26 committed:
Stylize Tundari Document

Bringing Tundari document into style compliance.

Of note, I have changed the Clans section header into a Tundari Clans chapter header, in order to make the header hierarchy of that section behave. If we prefer not to separate these into different chapters, we can instead leave Clans as a section and have both the clan and its traits be separate subsections a la 

	===== Clans =====
	
	== Rra’mau ==
	
	== Rra'mau Traits ==

Reveraine committed:
Merge pull request #291 from bleehu/Turtlelord26-Stylize_Tundari

Stylize Tundari Document

bleehu committed:
Merge branch 'dev' into MH-update-Engineer

bleehu committed:
removed many nanite references from Engineer Process Document

bleehu committed:
removed Nanites from Engineer

bleehu committed:
removed one more nanite reference from Engineering Processes

bleehu committed:
removed all of the mentions of nanites from Engineer Processes

Turtlelord26 committed:
Added poison weakness rule.

re issue #290

Turtlelord26 committed:
Ugh typo

Fed it to a starving reptile.

Turtlelord26 committed:
Further typography

Fed to a less-starving reptile.

Turtlelord26 committed:
Merge pull request #293 from bleehu/Turtlelord26-Tundari&Poison

Added poison weakness rule.

Turtlelord26 committed:
Implementing Movement in Microgravity

I decided for the sake of complexity to allow for a lot of GM Discretion for things like angles and collision physics. I've also focused almost entirely on movement, restricting the effects of weapons fire to increased difficulty of Anchoring oneself.

I've also hijacked the 02 slot for nonstandard environmental rules. This is of course subject to change on request.

Remember that this has 'Supplemental Rulebook Material' written all over it.

Turtlelord26 committed:
Review Notes

Turtlelord26 committed:
Merge pull request #294 from bleehu/Turtlelord26-Microgravity_Movement

Implementing Movement in Microgravity

bleehu committed:
added some shield and armor definitions to glossary

bleehu committed:
Revert "added some shield and armor definitions to glossary"

This reverts commit 5b96306495c05b1156f09a95014407bd9c0113b5.

bleehu committed:
Revert "Revert "added some shield and armor definitions to glossary""

This reverts commit 5194e40ff5ff9f8262024452096c8aebf1c76a62.

bleehu committed:
cleaned up grammar of glossary

bleehu committed:
some feat styling

bleehu committed:
Revert "some feat styling"

This reverts commit 84b9efd0308f465ab1da68a97790a6ddcb380c7e.

bleehu committed:
made updates as requested by Engineer review.

bleehu committed:
Merge pull request #301 from bleehu/MH-Issue-300-Armor-Glossary

added some shield and armor definitions to glossary

bleehu committed:
fixes that don't fix things aren't fixes

bleehu committed:
Merge pull request #292 from bleehu/MH-update-Engineer

Mh update engineer

bleehu committed:
bumped version number

Turtlelord26 reviewed (dismissed):
Changes Recognized by Github by File

# Basic Rules
-Style ✔️
-New Section on Reactions, Functional but basically codified houseruled content. ✔️
-Added definition of opportunity attack, functional codified houserule that has been tested ✔️
-Close-range prone nerf, functional, pretty sure not tested but I’ll approve it on aesthetic grounds ✔️
-Added rolling the average, functional, confident not tested, likely waiting on racial traits. ❌


# Conditions
-Introduced, functional. As it basically consists of states we've been houseruling in lieu of actual rules, I consider it tested. ✔️ 

# Skill DCs
-Style, nonfunctional, ✔️ 

# Microgravity Rules
-Exists, Functional, not yet tested and far too different to let through without playing them ❌ 

# Glossary
-Expanded and Styled, Nonfunctional ✔️

# New Player Walkthrough
-Exists, Nonfunctional change to the game but very functional change to player onboarding, not tested (or complete) ❌ 

# Tundari Rework
-Exists, Functional change, not yet Tested - cannot be tested before completion of species reworks since we seem reluctant to mix old-style and new-style species characters ❌ 

# Races
-I think it's all style, which is nonfunctional and approved ✔️. If there are functional bits that are escaping me atm I'm not super concerned since I have confidence in the species reworks in progress.
-Oh, cool, this is when we removed Archon. Functional, testing not applicable. ✔️

# Classes
-Style, nonfunctional ✔️
-Engineer reworked to use Energy, Material, Munitions. Very much a functional change. Require test feedback from @Reveraine ❔ 

# Feats
-Style, nonfunctional ✔️ 
-Last Stand feat moved from active to passive (& cost removed), functional but doesn't change the core of the feat and the power level seems right, so testing not required. ✔️ 

# Attachments
-Style, as above. ✔️ 
-Lanyard buffed with explicit quickdraw synergy, technically functional, I believe Geoff has used this without seeming unduly powerful. ✔️ 

# Melee & Thrown
-Style, as above, ✔️ 

# Pistols
-Style, as above. ✔️ 
-Damage buff to the Anvil R14, with price increase. Functional but small enough not to require explicit testing. I tracked it to commit #ac92fad5b9e083c32225f227f815c7d0df7f6d88, by 1source, who it is safe to assume knew what he was doing. ✔️ 
-Ammo size change to Anvil R14, nonfunctional ✔️ 

# SMG & Shotguns
-Style ✔️ 

# Carbines & Assault Rifles
-Style ✔️ 

# Long Rifles
-Style ✔️ 

# Heavy Weapons
-Style ✔️ 

# Armor
-Style ✔️ 
-Armor points -> Armor strength, nonfunctional ✔️ 

# New skills from Opossum
-Exists, functional, much testing and joy ✔️ 

# Old skills
-style ✔️ 

# Items
-Style ✔️ 
-Added replacement extruder tool, functional, untested to my knowledge but really the only question is the price point and that's going to be obsolete soon enough ✔️ 
-Added Hazmat Suit, tested as homebrew ✔️ 
-Added zipcuffs, not sure if tested but it's just a price-to-DC ratio question. ❔ 
-added glow stick, has not yet been used in test but they seem fine ✔️ 
-added nanite pack, functional, I do recall these being used, also they are gone later ✔️ 
-added marker, these have been purcashed, though not sure if used. ✔️ 

# Tactical Movement
-Style ✔️ 

# Doctor Procedures
-Style, ✔️
-Added APLs, Functional, not tested ❔
-Various procedures updated to combat 2.0, basically nonfunctional ✔️
-Harmacist Spike gains scaling, functional, probably tested, probably ok on power level ✔️
-Added whateverectomy, functional, not tested ❌

# Engineer Procedures
-Style, ✔️ 
-Edits corresponding to the resource rework approved above ✔️ 

# Hacking
-Style, ✔️

bleehu committed:
removed features not yet tested in RC before mass move to play in version bump to Beta 0.0.11
	modified:   01_01_Basic_Rules.txt
	deleted:    02_01_Advanced_Movement:_Microgravity.txt
	deleted:    CharacterCreation/00_New_Player_Walkthrough.txt
	deleted:    CharacterCreation/01_03_Races_Tundari
	modified:   CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

bleehu committed:
Merge branch 'play' into MH/RC-to-Play-pt2
